{
  "ID": "oval:org.debian:def:20084099",
  "Class": "vulnerability",
  "Metadata": {
    "Title": "CVE-2008-4099",
    "AffectedList": [
      {
        "Family": "unix",
        "Platform": "Debian GNU/Linux 9",
        "Product": "python-dns"
      }
    ],
    "References": [
      {
        "Source": "CVE",
        "RefID": "CVE-2008-4099",
        "RefURL": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2008-4099"
      }
    ],
    "Description": "PyDNS (aka python-dns) before 2.3.1-4 in Debian GNU/Linux does not use random source ports or transaction IDs for DNS requests, which makes it easier for remote attackers to spoof DNS responses, a different vulnerability than CVE-2008-1447.",
    "Debian": {
      "MoreInfo": ""
    }
  },
  "Criteria": {
    "Operator": "AND",
    "Criterias": [
      {
        "Operator": "OR",
        "Criterias": [
          {
            "Operator": "AND",
            "Criterias": null,
            "Criterions": [
              {
                "Negate": false,
                "TestRef": "oval:org.debian.oval:tst:2",
                "Comment": "all architecture"
              },
              {
                "Negate": false,
                "TestRef": "oval:org.debian.oval:tst:2748",
                "Comment": "python-dns DPKG is earlier than 2.3.1-5"
              }
            ]
          }
        ],
        "Criterions": null
      }
    ],
    "Criterions": [
      {
        "Negate": false,
        "TestRef": "oval:org.debian.oval:tst:1",
        "Comment": "Debian 9 is installed"
      }
    ]
  }
}